include/z64actor.h

```diff
@@ -355,6 +355,103 @@ typedef enum {
     DOORLOCK_NORMAL_SPIRIT
 } DoorLockType;
 
+typedef enum {
+    /* 0x00 */ NAVI_ENEMY_DEFAULT,
+    /* 0x01 */ NAVI_ENEMY_GOHMA,
+    /* 0x02 */ NAVI_ENEMY_GOHMA_EGG,
+    /* 0x03 */ NAVI_ENEMY_GOHMA_LARVA,
+    /* 0x04 */ NAVI_ENEMY_SKULLTULA,
+    /* 0x05 */ NAVI_ENEMY_BIG_SKULLTULA,
+    /* 0x06 */ NAVI_ENEMY_TAILPASARAN,
+    /* 0x07 */ NAVI_ENEMY_DEKU_BABA,
+    /* 0x08 */ NAVI_ENEMY_BIG_DEKU_BABA,
+    /* 0x09 */ NAVI_ENEMY_WITHERED_DEKU_BABA,
+    /* 0x0A */ NAVI_ENEMY_DEKU_SCRUB,
+    /* 0x0B */ NAVI_ENEMY_UNUSED11,
+    /* 0x0C */ NAVI_ENEMY_KING_DODONGO,
+    /* 0x0D */ NAVI_ENEMY_DODONGO,
+    /* 0x0E */ NAVI_ENEMY_BABY_DODONGO,
+    /* 0x0F */ NAVI_ENEMY_LIZALFOS,
+    /* 0x10 */ NAVI_ENEMY_DINOLFOS,
+    /* 0x11 */ NAVI_ENEMY_FIRE_KEESE,
+    /* 0x12 */ NAVI_ENEMY_KEESE,
+    /* 0x13 */ NAVI_ENEMY_ARMOS,
+    /* 0x14 */ NAVI_ENEMY_BARINADE,
+    /* 0x15 */ NAVI_ENEMY_PARASITIC_TENTACLE,
+    /* 0x16 */ NAVI_ENEMY_SHABOM,
+    /* 0x17 */ NAVI_ENEMY_BIRI,
+    /* 0x18 */ NAVI_ENEMY_BARI,
+    /* 0x19 */ NAVI_ENEMY_STINGER,
+    /* 0x1A */ NAVI_ENEMY_PHANTOM_GANON_2,
+    /* 0x1B */ NAVI_ENEMY_STALFOS,
+    /* 0x1C */ NAVI_ENEMY_BLUE_BUBBLE,
+    /* 0x1D */ NAVI_ENEMY_WHITE_BUBBLE,
+    /* 0x1E */ NAVI_ENEMY_GREEN_BUBBLE,
+    /* 0x1F */ NAVI_ENEMY_SKULLWALLTULA,
+    /* 0x20 */ NAVI_ENEMY_GOLD_SKULLTULA,
+    /* 0x21 */ NAVI_ENEMY_VOLVAGIA,
+    /* 0x22 */ NAVI_ENEMY_FLARE_DANCER,
+    /* 0x23 */ NAVI_ENEMY_TORCH_SLUG,
+    /* 0x24 */ NAVI_ENEMY_RED_BUBBLE,
+    /* 0x25 */ NAVI_ENEMY_MORPHA,
+    /* 0x26 */ NAVI_ENEMY_DARK_LINK,
+    /* 0x27 */ NAVI_ENEMY_SHELL_BLADE,
+    /* 0x28 */ NAVI_ENEMY_SPIKE,
+    /* 0x29 */ NAVI_ENEMY_BONGO_BONGO,
+    /* 0x2A */ NAVI_ENEMY_REDEAD,
+    /* 0x2B */ NAVI_ENEMY_PHANTOM_GANON_1,
+    /* 0x2C */ NAVI_ENEMY_UNUSED2C,
+    /* 0x2D */ NAVI_ENEMY_GIBDO,
+    /* 0x2E */ NAVI_ENEMY_DEAD_HANDS_HAND,
+    /* 0x2F */ NAVI_ENEMY_DEAD_HAND,
+    /* 0x30 */ NAVI_ENEMY_WALLMASTER,
+    /* 0x31 */ NAVI_ENEMY_FLOORMASTER,
+    /* 0x32 */ NAVI_ENEMY_TWINROVA_KOUME,
+    /* 0x33 */ NAVI_ENEMY_TWINROVA_KOTAKE,
+    /* 0x34 */ NAVI_ENEMY_IRON_KNUCKLE_MINIBOSS,
+    /* 0x35 */ NAVI_ENEMY_IRON_KNUCKLE,
+    /* 0x36 */ NAVI_ENEMY_SKULL_KID_ADULT,
+    /* 0x37 */ NAVI_ENEMY_LIKE_LIKE,
+    /* 0x38 */ NAVI_ENEMY_UNUSED56,
+    /* 0x39 */ NAVI_ENEMY_BEAMOS,
+    /* 0x3A */ NAVI_ENEMY_ANUBIS,
+    /* 0x3B */ NAVI_ENEMY_FREEZARD,
```

Comment:
```suggestion
    /* 0x3B */ NAVI_ENEMY_FREEZZARD,
```
:YEP:

Reply:
I got curious and looked this up, apparently it's a typo in OoT: MM and every other game spells it with one 'z'. Even a few of the strategy guides spell it with one.

Reply:
Oops. To be clear I think this typo in the oot text probably shouldn't be reflected in the enum